lib/rules/autocomplete-matches.js

```diff
@@ -0,0 +1,45 @@
+const { text, aria, dom } = axe.commons;
+
+const autocomplete = node.getAttribute('autocomplete');
+if (!autocomplete || text.sanitize(autocomplete) === '') {
+	return false;
```

Comment:
I think this line of code negates the comment asking you to add check for the case where `autocomplete` is empty or missing.

Reply:
It should, but in the case of a custom rule someone might have messed with this. The check should handle it too.